### ISSUE TITLE
Swap out \begin{center} for \centering

### ISSUE THREAD

**Issue by prowlett** [prowlett/nimsticks#4]
Because it doesn't add vertical space and because it doesn't work in standalone. 

Change `\nimstick` definition to 

```
\newcommand{\nimgame}[2][block]{%
    \ifthenelse{\equal{#1}{inline}}{}{{\centering}%
        \def\listofgames{#2}%
        \foreach \heap in \listofgames {%
            \foreach \heapindex in {1, ..., \heap} {%
                \drawnimstick\hspace{0.5mm}%
            }%
            \hspace{10mm}%
        }%
        \ifthenelse{\equal{#1}{inline}}{}{\par}}%
}
```

This means the command can be called as `\nimgame{a,b,c}` or `\nimgame[inline]{a,b,c}` as before except both now work in a `standalone` document and there is no extra spacing around the centred one.

**Comment by prowlett** [comment]
Actually it looks like this should be 

```
\newcommand{\nimgame}[2][block]{%
    \ifthenelse{\equal{#1}{inline}}{}{\bgroup\centering}%
        \def\listofgames{#2}%
        \foreach \heap in \listofgames {%
            \foreach \heapindex in {1, ..., \heap} {%
                \drawnimstick\hspace{0.5mm}%
            }%
            \hspace{10mm}%
        }%
    \ifthenelse{\equal{#1}{inline}}{}{\par\egroup}%
}
```

**Comment by prowlett** [comment]
Fixed in fe4f04d
